### PR TITLE
Added multilib for modules

### PR DESCRIPTION
**Pull request by Korulag**

### CODE REVIEW

File: alws/utils/multilib.py [hunk 1]
```diff
@@ -19,6 +24,252 @@
...
+            endpoint = (f'api/v1/distros/{ref_name}/{ref_ver}/module/'
+                        f'{module_name}/{module_stream}/x86_64/')
+            response = await self.__call_beholder(endpoint)
+            devel_packages = await self.parse_response(query, response)
+            packages += devel_packages
+            # Deduplicate packages
+            packages_mapping = {f'{i["name"]}-{i["version"]}': i
+                                for i in packages}
+            packages = list(packages_mapping.values())
+        logging.info('Found something: %s', str(packages))
```
Comment (anfimovdm) [inline alws/utils/multilib.py:142]: Found packages would be sounds more clear

File: alws/utils/multilib.py [hunk 2]
```diff
@@ -19,6 +24,252 @@
...
+        if self._module_index and self._module_index.has_devel_module():
+            module_name = f'{module_name}-devel'
+            endpoint = (f'api/v1/distros/{ref_name}/{ref_ver}/module/'
+                        f'{module_name}/{module_stream}/x86_64/')
+            response = await self.__call_beholder(endpoint)
+            devel_packages = await self.parse_response(query, response)
+            packages += devel_packages
+            # Deduplicate packages
+            packages_mapping = {f'{i["name"]}-{i["version"]}': i
+                                for i in packages}
```
Comment (anfimovdm) [inline alws/utils/multilib.py:140]: can we lose some packages if regular and devel package have same name and version?

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/utils/rpm_package.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+from alws.utils.pulp_client import PulpClient
+
+
+__all__ = ['get_rpm_package_info']
+
+
+async def get_rpm_package_info(pulp_client: PulpClient, rpm_href: str,
+                               include_fields: typing.List[str] = None,
+                               exclude_fields: typing.List[str] = None) \
+        -> (str, dict):
```
[the remark this answers is not in the document]
Comment (Korulag) [reply alws/utils/rpm_package.py:12]: This one is needed for creating a dictionary out of results in format {'href': {"name": "$pkg_name"...}}

File: alws/utils/multilib.py (hunk 2 above)
Comment (Korulag) [reply alws/utils/multilib.py:140]: The packages in repo are the same. The only difference is where they will be put inside modules metadata and there everything seems to be working great

File: alws/utils/rpm_package.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (anfimovdm) [reply alws/utils/rpm_package.py:12]: i've meant about codestyle

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**